docroot/modules/custom/foia_webform/src/FoiaSubmissionEmailService.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\QueueWorker;
+
+use Drupal\foia_webform\AgencyLookupServiceInterface;
+use Drupal\webform\WebformSubmissionInterface;
+use Drupal\node\Entity\Node;
+
+/**
+ * Class FoiaSubmissionEmailService.
+ *
+ * @package Drupal\foia_webform\Plugin\QueueWorker
+ */
+class FoiaSubmissionEmailService implements FoiaSubmissionServiceInterface {
```

Comment:
No need to include this code with this PR. Let's keep the PR as relevant to issue 121 as possible.